safehttp/header_test.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package safehttp
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestSet(t *testing.T) {
+	h := newHeader(http.Header{})
+	if err := h.Set("pizza-pasta", "potato-carrot"); err != nil {
```

Comment:
Can we make this test clearer with some better key/value names? e.g. `fooKey` and `barValue`?

Reply:
Header-Key-Pizza: header value Margherita;